docs/_packages/web_api.md

```diff
@@ -246,7 +246,7 @@ pattern for people that use _functional programming_.
 ---
 
 ### Opening modals
-[Modals](https://api.slack.com/block-kit/surfaces/modals) can be created by calling the `views.open` method. The method requires you to pass a valid [view payload](https://api.slack.com/reference/block-kit/views) in addition to a `trigger_id`, which can be obtained when a user invokes your app using a slash command, clicking a button, or using [another interactive action](https://api.slack.com/reference/messaging/interactive-components).
+[Modals](https://api.slack.com/block-kit/surfaces/modals) can be created by calling the `views.open` method. The method requires you to pass a valid [view payload](https://api.slack.com/reference/block-kit/views). Additionally, for apps not built with the CLI, the  `trigger_id`, which can be obtained in the payload returned when a user invokes your app using a slash command, clicking a button, or using [another interactive action](https://api.slack.com/reference/messaging/interactive-components). For apps built with the CLI, an `interactivity_pointer` is required instead of a `trigger_id`, which can be obtained in the `interactivity` object within the payload returned when a user invokes your app using a slash command, clicking a button, or using [another interactive action](https://api.slack.com/reference/messaging/interactive-components).
```

Comment:
How about something along the lines of the below? 

### Opening modals
[Modals](https://api.slack.com/block-kit/surfaces/modals) can be created by calling the `views.open` method. The method requires you to pass a valid [view payload](https://api.slack.com/reference/block-kit/views).

#### Handling modals during Slack Function handling (beta)
 If you are trying to open or update a modal from a Slack Function interactivity handler {link to Interactivity page here}, pass the `interactivity_pointer` you received from your event payload in your `views.open` method. 
 
```js
// Code snippet here
```

Reply:
I also feel like we ought to have some version of this in the Bolt documentation as well!